src/script/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+# Must be included before CMAKE_INSTALL_INCLUDEDIR is used.
+include(GNUInstallDirs)
+
+add_library(bitcoinconsensus
+  bitcoinconsensus.cpp
+  $<TARGET_OBJECTS:bitcoin_consensus>
+  $<TARGET_OBJECTS:bitcoin_crypto>
+  ../support/cleanse.cpp
+)
+target_compile_definitions(bitcoinconsensus
+  PRIVATE
+    BUILD_BITCOIN_INTERNAL
+)
+target_include_directories(bitcoinconsensus
+  PUBLIC
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+target_link_libraries(bitcoinconsensus
+  PRIVATE
+    core
+    secp256k1
+    $<TARGET_NAME_IF_EXISTS:link_ssp>
+)
+set_target_properties(bitcoinconsensus PROPERTIES
+  SOVERSION 0
+  VERSION 0.0.0
+)
+
+install(TARGETS bitcoinconsensus
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+install(FILES bitcoinconsensus.h
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+)
+
+include(GeneratePkgConfigFile)
+generate_pkg_config_file(${PROJECT_SOURCE_DIR}/libbitcoinconsensus.pc.in libbitcoinconsensus.pc)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libbitcoinconsensus.pc
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
+)
+
+function(make_bitcoinconsensus_dll_available target)
+  if(WIN32 AND BUILD_SHARED_LIBS)
+    # The DLL must reside either in the same folder where the executable is
```

Comment:
> The DLL must reside either in the same folder where the executable is
> or somewhere in PATH. Using the former option.

Why? Is this some CMake behaviour? Would be good to be more clear in this comment.

Reply:
This is a  [Windows](https://learn.microsoft.com/en-us/troubleshoot/windows-client/deployment/dynamic-link-library)-specific behavior:
> ... the Windows operating system searches for ... DLL in the following locations in this order:
>
> 1. The application folder
>
> 2. The current folder
>
> 3. The Windows system folder
>
> 4. The Windows folder